vcr/signature/signature.go

```diff
@@ -0,0 +1,51 @@
+package signature
+
+import (
+	"encoding/json"
+	ssi "github.com/nuts-foundation/go-did"
+	"github.com/nuts-foundation/nuts-node/crypto"
+	"github.com/nuts-foundation/nuts-node/crypto/hash"
+)
+
+// Suite is an interface which defines the methods a signature suite implementation should implement.
```

Comment:
How do "signature suites" and "proof types" relate? Do we really need the 2 concepts or can there just be 1?

Reply:
Must be multiple. We can have a JSONLD or a JWT proof with JsonWebSignature, or Ed25519Signature2018 or RsaSignature2018 suites.